### DISCUSSION TITLE
pyAEDT Simulation Automation in AEDT Icepak

### DISCUSSION THREAD

**Discussion by RishabhSehgal** [ansys/pyaedt#3213] [Q&A] [answered]
I was going through online tutorials and user guides for scripting using pyAEDT particularly for Icepak. I came across this video for which I have provided the link here. I still had few questions in the automation script. Is it possible to get the script that is used in this tutorial?

[pyAEDT Simulation Automation in AEDT Icepak - YouTube](https://www.youtube.com/embed/xEL8ZVgsJs8)

Thanks

Best Regards

Rishabh

**Comment by Samuelopez-ansys** [comment, marked as answer]
@RishabhSehgal  I don't have this specific script, but in the example website, you will find some Icepak examples:
https://aedt.docs.pyansys.com/version/stable/examples/04-Icepak/index.html
